### PR TITLE
feat: add landing page's fourth section

### PR DESCRIPTION
**Pull request by hsglc**

### CODE REVIEW

File: gurubu-client/src/app/styles/page/style.scss [hunk 1]
```diff
@@ -4,3 +4,10 @@
 // Sections
 @import "./hero-section.scss";
 @import "./testimonial.scss";
+@import "./pricing.scss";
+
+main {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
```
Comment (armagandalkiran) [inline gurubu-client/src/app/styles/page/style.scss:13]: why did you add this code block ?

Comment (hsglc) [reply gurubu-client/src/app/styles/page/style.scss:13]: That was already there, I didn't add that. Could you check it out again please?

Comment (armagandalkiran) [reply gurubu-client/src/app/styles/page/style.scss:13]: I think linter issue ok then LGTM for me I will merge it!

**Review by armagandalkiran — APPROVED** [review]

**Comment by armagandalkiran** [comment]
Thank you for your contribution! 🚀🤘

**Comment by mertcanaltin** [comment]
congrats 🎊 @hsglc
